dashboard-app/src/components/TemplateOnePages/ReportPage.js

```diff
@@ -40,7 +79,8 @@ const ReportPage = (props) => {
     <div>
         <Header title={title}/>
         {objectList.map(obj =>
-            <InformationSection key={obj.id} type={obj.type}/>
+            // Added a 'dataid' prop to pass to chart components ('key' is not accessable) so that the divs can have unique IDs
+            <InformationSection key={obj.id} dataid={obj.id} type={obj.type}/>
```

Comment:
Looks like you can just get rid of key here then

Reply:
Whenever you map to make new components, React likes to have a key prop to differentiate them if one gets removed or created